emacs/README.md

```diff
@@ -0,0 +1,69 @@
+# zeek-language-server

```

Comment:
```suggestion
# Emacs setup for zeek-language-server

```

Let's also link this from the top-level README.

Since this directory contains only a README file for now, I am wondering whether just adding a top-level `README.emacs` would be sufficient.

Reply:
I'd be alright with just moving it up.